examples/pong.qasm

```diff
@@ -41,7 +41,7 @@ delay[s] q[1];
 h q[1];
 measure q[1] -> c[0];
 delay[s] q[1];
-boxto[maxduration] { // fixed-duration anonymous subroutine
+boxto[maxdur] { // fixed-duration anonymous subroutine
```

Comment:
I think `boxto` is just `box` now. @levbishop, can you confirm?

Reply:
Addressed in https://github.com/Qiskit/openqasm/pull/204/commits/f56961122910730ee52c6b091dc1cfedfc28986b.